src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,9 +37,27 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(trimmedAddress)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
+        createAddress(trimmedAddress);
         this.value = trimmedAddress;
     }
 
+    /**
+     * Initialises the following attributes: block, postalCode, street, unit.
```

Comment:
`Initialises` is a very general term, may be `Splits addresses into ...` is better.